src/sciline/pipeline.py

```diff
@@ -96,6 +241,8 @@ def __init__(
         """
         self._providers: Dict[Key, Provider] = {}
         self._subproviders: Dict[type, Dict[Tuple[Key | TypeVar, ...], Provider]] = {}
+        self._param_tables: Dict[Key, ParamTable] = {}
+        self._param_series: Dict[Key, Key] = {}
```

Comment:
Confusing name. This maps from parameter names to dimensions., not so series objects.